app/views/templates/package_action.erb

```diff
@@ -26,7 +26,17 @@ template_inputs:
   advanced: true
 %>
 
-die() {
+<%
+  supported_families = ['Redhat', 'Debian']
+  render_error(N_('Unsupported or no operating system found for this host.')) unless @host.operatingsystem && supported_families.include?(@host.operatingsystem.family)
+%>
+
+# Helper function that exits with a particular message and code.
+#
+# Usage:
+#   exit_with_message 2 "Could not do a thing"
+#
+exit_with_message() {
```

Comment:
Wouldn't it be even more user friendly to use the `function exit_with_message()` syntax?